src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,165 @@
+        System.out.println(Files.readAllLines(file).size() + " lines");
+        System.out.println(countWords(Files.readAllLines(file)) + " words");
+        Calendar calendar = Calendar.getInstance();
+        BasicFileAttributes attributes = Files.readAttributes(file, BasicFileAttributes.class);
+        calendar.setTimeInMillis(attributes.lastModifiedTime().toMillis());
+
+        System.out.println(calendar.getTime() + " last modified");
+        System.out.println("Size: " + attributes.size() + " bytes");
+    }
+
+    private long countWords(List<String> lines) {
+        //this is a hw for lesson 5, so students are not obliged to use Stream API
+        return lines.stream()
+                .map(line -> Stream.of(line.split(" ")))
+                .flatMap(o -> o)
+                .filter(str -> str.matches("^[a-zA-Z]+"))
+                .count();
+    }
+
+    private void help() {
+        for (String info : commandInfo.values()) {
+            System.out.println(info);
+        }
+    }
+
+    private void help(String commandName) {
+        System.out.println(commandInfo.get(commandName));
+    }
+
+    private void saveText(String input) throws IOException {
+        if (input.isEmpty()) {
+            return;
+        }
+        System.out.println("Save text? Y/N");
+        if (askYesNo()) {
+            String[] saveFilePath = getSaveFilePath();
+            create(saveFilePath[0], saveFilePath[1]);
+            writeToFile(saveFilePath[0], saveFilePath[1], input);
+        }
+    }
+
+    private boolean askYesNo() {
+        Scanner scanner = new Scanner(System.in);
+        String answer = scanner.next();
+        switch (answer) {
+            case "Y":
+                return true;
+            case "N":
+                return false;
+            default:
+                System.out.println("Unknown answer. Please, write Y or N");
+                askYesNo();
+        }
+        return false;
+    }
+
+    private void writeToFile(String path, String fileName, String input)
+            throws FileNotFoundException {
+        try (PrintWriter writer = new PrintWriter(path + File.separator + fileName)) {
+            writer.write(input);
+        }
+    }
+
+    private String[] getSaveFilePath() {
+        System.out.println("Please, write path and name for the new text file:");
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+        return input.split(" ");
+    }
+
+    private void read(String path, String fileName) {
+        try (BufferedReader reader = new BufferedReader(
+                new FileReader(path + File.separator + fileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                System.out.println(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            System.out.println("Warning: no such file or directory");
```

Comment:
IOException can happen for many other reasons. We should check if file exists if we want to cover this case. In all other cases - returhow exception.